backend/runners/load_addresses.py

```diff
@@ -1,28 +1,41 @@
 """Runner to load addresses to databse from a CSV."""
+import argparse
 import csv
 
-from backend.lib.database.postgres import connect
-from backend.lib.database.postgres import methods
+from backend.lib.database.postgres import connect, methods, postgis
 from backend.lib.database.tables import address
 
-CSV_FILE = './data/last-1100-geocoded-pointBs-93710.csv'
+
+def _get_arguments():
+    """Build argument parser."""
+    parser = argparse.ArgumentParser(description="""
+        This script loads provider address data from a specified file into PostGIS.
+    """)
+    parser.add_argument(
+        '-f', '--filepath',
+        help='CSV filepath containing provider address data.',
+        required=True,
+        type=str
+    )
+    return parser.parse_args().__dict__
 
 
-def load_csv(csv_file=CSV_FILE):
+def load_csv(kwargs):
```

Comment:
maybe use `arguments` instead as a variable?

Reply:
Or use `filepath` as argument and do `load_csv(_get_arguments())`

Reply:
Done.